src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Why do we need 2 Color enums? Lets remove this one

Reply:
I don't know where you saw Colors? I did change two day ago. I changed Colors on Color according you requirement

Reply:
![image](https://user-images.githubusercontent.com/117547096/209828068-1feae190-4c83-451b-8f7c-9f9ff9173d73.png)

Reply:
remove it